### PR TITLE
Pr tiltrotor sim

### PR DESCRIPTION
**Pull request by RomanBapst**
This brings in support for a quad tiltrotor in gazebo.
The model works, it needs a review and more testing.

### CODE REVIEW

File: ROMFS/sitl/mixers/tiltrotor_sitl.main.mix [hunk 1]
```diff
@@ -0,0 +1,41 @@
+Mixer for quad tiltrotor (x motor configuration)
+================================================
+
+This file defines a single mixer for a tiltrotor (SITL gazebo) with motors in X configuration. The plane has two ailerons and one elevator.
```
Comment (dagar) [inline ROMFS/sitl/mixers/tiltrotor_sitl.main.mix]: Add as a generic tiltrotor mixer?

**Comment by dagar** [comment]
@PX4BuildBot rebase

**Comment by dagar** [comment]
CI was full, rebasing again to kick it off. @PX4BuildBot rebase

**Comment by dagar** [comment]
@RomanBapst CI style failure. http://ci.px4.io:8080/blue/organizations/jenkins/Firmware/detail/PR-8936/1/pipeline/720

Locall you can install astyle and run `make check_format` or `make format`.

image

**Comment by RomanBapst** [comment]
@dagar Thanks! I fixed code style and added a pre-commit hook.

**Pull request merged**